CONTRIBUTING.md

```diff
@@ -31,3 +31,9 @@
 1. Update `CHANGELOG.md`.
    - Add the new section with the release version in the title.
    - Leave the `main` section empty (without any changelog entries).
+
+1. If `[EAS BUILD API]` tag is present in CHANGELOG
+   - Find version of the package `@expo/eas-build-job` used in the eas-cli.
+   - Find `Publish` commit in [expo/eas-build](https://github.com/expo/eas-build/commit/) for that version of `@expo/eas-build-job`. In most cases it will be in the latest publish commit.
+   - Find version of the package `eas-cli-local-build-plugin` published in that commit.
```

Comment:
```suggestion
   - Find the `eas-cli-local-build-plugin` version published in that commit.
```

Reply:
<img width="702" alt="Screenshot 2021-05-14 at 16 27 34" src="https://user-images.githubusercontent.com/5256730/118285148-74763300-b4d1-11eb-979a-0f8eaade02cb.png">
Maybe add this screen to make the instructions easier to follow.